Day1/day1_p1.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const arr = [1721, 979, 366, 299, 675, 1456];
+
+const searchTwoNumbers = arr => {
+  for (let i = 0; i <= arr.length; i++) {
+    for (let j = i + 1; j <= arr.length; j++) {
+      if (arr[i] + arr[j] === 2020) {
+        const a = arr[i];
+        const b = arr[j];
+        const array = [a, b];
+        return array;
+      }
+    }
+  }
+};
+
+searchTwoNumbers(arr);
```

Comment:
Why do you run this code here with the test data?
When you import it in any other place, this function will be executed and the result will be ignored.
For example, in your test, the function will be executed twice: the first time when you `require` the module, the second time when you use it in the test.